src/main/java/edu/kis/powp/jobs2d/command/history/CommandHistory.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.command.history;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import javax.swing.DefaultListModel;
+
+public class CommandHistory {
+
+    static DefaultListModel<HistoryEntry> entryHistoryList = new DefaultListModel<>();
```

Comment:
It would be safer to provide a read-only view through the getter

Reply:
Done